### PR TITLE
feat: vocab extensions

### PR DESCRIPTION
**Pull request by mcz-sz**

### CODE REVIEW

**Review by niklasbeinghaus — APPROVED** [review]

File: plat-vale/styles/Vocab/Platform/accept.txt [hunk 1]
```diff
@@ -34,9 +34,13 @@ Terragrunt
 Ubuntu
 YYYY
 [Aa]lternatives
+[Aa]utoscaler
+[Aa]utoscaling
```
Comment (niklasbeinghaus) [inline plat-vale/styles/Vocab/Platform/accept.txt:38]:
```suggestion
[Aa]utoscal(ing|er)
```

File: plat-vale/styles/Vocab/Platform/accept.txt [hunk 2]
```diff
@@ -48,22 +52,23 @@ YYYY
...
+ENV
 fluxcomp
 kubeconform
-kustomize
 macOS
 maintain
 mkdocs
 namespace
+repo
 repos
```
Comment (niklasbeinghaus) [inline plat-vale/styles/Vocab/Platform/accept.txt:73]:
```suggestion
repositor(ies|y|s)?
```

File: plat-vale/styles/Vocab/Platform/accept.txt (hunk 1 above)
Comment (mcz-sz) [reply plat-vale/styles/Vocab/Platform/accept.txt]: added

File: plat-vale/styles/Vocab/Platform/accept.txt (hunk 2 above)
Comment (mcz-sz) [reply plat-vale/styles/Vocab/Platform/accept.txt]: I removed repo and repos entirely. repository and its variations are already accepted by vale

**Pull request merged**